websocket_server/app.js

```diff
@@ -27,4 +29,8 @@ app.get('/metrics', async (req, res) => {
 	res.end(metrics)
 })
 
+app.get('/system-overview', (req, res) => {
+	res.json(getSystemOverview(rooms))
+})
```

Comment:
Can we expose those also as prometheus metrics in the /metrics endpoint instead? If there is something we want to expose separately it should also be limited to requests where the `METRICS_TOKEN` is passed